core/src/main/java/org/elasticsearch/index/search/MatchQuery.java

```diff
@@ -386,7 +385,7 @@ protected final Query createFieldQuery(TokenStream source, BooleanClause.Occur o
                     }
 
                     int positionLength = posLenAtt.getPositionLength();
-                    if (!isGraph && positionLength > 1 && ((flagsAtt.getFlags() & GRAPH_FLAG) == GRAPH_FLAG)) {
+                    if (positionLength > 1) {
```

Comment:
Don't need `TypeAttribute` and if LUCENE-7603 is here we can remove all the Overrides on `QueryBuilder` that were added for graph support.